### PR TITLE
Dilpreet | Stage | For bank imported transaction entry pop up should not get overlap with account dropdown on ledger

### PR DESCRIPTION
**Pull request by dilpreetsinghofficial**
https://app.clickup.com/t/2mkqera

### CODE REVIEW

**Comment by dilpreetsinghofficial** [comment]
REVIEWED

**Review by the-prateek-mishra — APPROVED** [review]

File: apps/web-giddh/src/app/material-ledger/ledger.component.html [hunk 1]
```diff
@@ -920,16 +946,36 @@ <h3 class="ledger-main-head">
...
+                                                                    >
+                                                                </div>
+                                                                <div
+                                                                    *ngIf="txn.showAccountDropdown"
+                                                                    class="w-100"
+                                                                    (click)="txn.showDropdown = true"
+                                                                >
+                                                                    <span
+                                                                        [ngClass]="{
+                                                                            'account-placeholder no': true,
```
Comment (the-prateek-mishra) [inline apps/web-giddh/src/app/material-ledger/ledger.component.html]: Please check class name
```suggestion
                                                                            'account-placeholder-no': true,
```

File: apps/web-giddh/src/app/material-ledger/ledger.component.html [hunk 2]
```diff
@@ -920,16 +946,36 @@ <h3 class="ledger-main-head">
...
-                                                                        showBankLedgerPopup(txn, item)
+                                                                        showBankLedgerPopup(txn, item);
+                                                                        txn.showAccountDropdown = true
                                                                     "
                                                                 >
                                                                     <span
                                                                         [ngClass]="{
-                                                                            'account-placeholder': true,
-                                                                            'account-placeholder-text':
+                                                                            'account-placeholder yes': true,
```
Comment (the-prateek-mishra) [inline apps/web-giddh/src/app/material-ledger/ledger.component.html]: Please check class name
```suggestion
                                                                            'account-placeholder-yes': true,
```

File: apps/web-giddh/src/app/material-ledger/ledger.component.html [hunk 3]
```diff
@@ -658,6 +658,29 @@ <h3 class="ledger-main-head">
...
+                                                                    *ngIf="!txn.showAccountDropdown"
+                                                                    class="w-100"
+                                                                    (click)="
+                                                                        showBankLedgerPopup(txn, item);
+                                                                        txn.showAccountDropdown = true
+                                                                    "
+                                                                >
+                                                                    <span
+                                                                        [ngClass]="{
+                                                                            'account-placeholder yes': true,
```
Comment (the-prateek-mishra) [inline apps/web-giddh/src/app/material-ledger/ledger.component.html]: Please check class name
```suggestion
                                                                            'account-placeholder-yes': true,
```

**Review by the-prateek-mishra — CHANGES_REQUESTED** [review]

**Review by the-prateek-mishra — APPROVED** [review]

**Comment by dilpreetsinghofficial** [comment]
reviewed

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**